average.js

```diff
@@ -1,6 +1,7 @@
 
 function average(numbers) {
-  return numbers.reduce((p, c)=> p + c, 0) / numbers.length;
+  const newNumbers = numbers.filter(val => !Number.isNaN(val))
```

Comment:
Try a more descriptive name for `newNumbers` - the purpose of naming variables is to recognize their intent without having to read the entire code.